test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
Ooops, this one fell through the cracks of github :P

I agree with the new concept of `_execute` and `execute`, but I definitely think that it should be a separate PR as this one is already starting to overflow its scope a bit much